misc/v3-task-endpoint-validator-windows/v3-task-endpoint-validator-windows.go

```diff
@@ -41,6 +41,7 @@ type TaskResponse struct {
 	PullStartedAt      *time.Time          `json:"PullStartedAt,omitempty"`
 	PullStoppedAt      *time.Time          `json:"PullStoppedAt,omitempty"`
 	ExecutionStoppedAt *time.Time          `json:"ExecutionStoppedAt,omitempty"`
+	//AvailabilityZone   string              `json:"AvailabilityZone"`
```

Comment:
Why is this commented out?

Reply:
will update it, haven't add it to windows yet.